packages/editor/src/data/data.ts

```diff
@@ -21,14 +22,17 @@ const findComponent = (
   if (id.startsWith(LAYOUT_DROPZONE_ID_PREFIX)) {
     return findLayoutComponent(data, id.replace(LAYOUT_DROPZONE_ID_PREFIX, ''));
   }
+  if (id.startsWith(TABLE_DROPZONE_ID_PREFIX)) {
+    return findTableComponent(data, id.replace(TABLE_DROPZONE_ID_PREFIX, ''));
+  }
   return findComponentDeep(data, id, parent);
```

Comment:
instead of mix this table component stuff with the normal data behaviour, I just had an idea...
What about create an own DndContext just for the datatable columns. Then you maybe don't have to handle stuff like that something else can be dropped into the table or that a column can be dropped outside...

Reply:
Thats a good idea 👍 would it be possible to try this in a later PR. Maybe all this stuff will change anyway as i implement in the next PR the updated/improved form.ts

Reply:
-> new subtask